app/views/shared/moj_components_templates/_interruption_card_template.html.erb

```diff
@@ -3,7 +3,16 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l moj-interruption-card__heading"><%= heading %></h1>
-        <%= body %>
+        <% if body.present? %>
+        <div class="govuk-body moj-interruption-card__body">
+          <%= body %>
+        </div>
+        <% end %>
+        <% if actions.present? %>
+        <div class="govuk-button-group moj-interruption-card__actions">
+          <%= actions.call %>
+        </div>
+        <% end %>
```

Comment:
Nit: indentation seems incorrect.

Reply:
This was considered. Indenting here actually indents the divs in the markup an extra unneeded level when they are rendered which did not seem appropriate. I went with removing unneeded indent in markup  but don't mind either way - unneeded indent in rendered markup, or no indent in unrendered erb? Your call.